cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,44 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+=====================
+Application Logbacks
+=====================
+
+YARN containers spawned by a CDAP application use the default container logback, `logback-container.xml`,
+packaged with CDAP. This `logback.xml` does log rotation and depending on the use case, may be sufficient.
+However, it is also possible to specify a `logback.xml` for a CDAP application.
+The packaged `logback.xml` is then used for each container spawned by a program of the application.
+
+Writing and packaging a logback.xml
+===================================
+To specify a `logback.xml` to be used by the application, it is sufficient to package the file as a resource in the jar.
+The `pom.xml` of the application may look like this, with the `resources` directory containing the `logback.xml`::
```

Comment:
The `pom.xml` of the application, with the application's `resources` directory containing the `logback.xml`, might look like this::